### ISSUE TITLE
Code - (firefox/privacy/products) Protection Report Landing Page

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7709]
## Description

@stephaniehobson will code a new Privacy Hub page at e.g. /firefox/privacy, and the protection report landing page will be the 2nd page in the Hub. 

This card is about building the Protection Report landing page; work with @stephaniehobson on that.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] Protection report page is integrated into new privacy hub
- [ ] It is launched (or ready to prod push on a later date) by 2019-10-11
- [x] It includes the multi-state CTA specified in copy docs
- [x] Lives at this URL - mozilla.org/firefox/privacy/products

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] new firefox navigation being complete

#### Tasks
- [x] Code it with conditionals
- [x] Confirm that about:protections can be linked to in the page to open in the browser

**Comment by alexgibson** [comment]
WIP sketch file is linked in https://github.com/mozilla/bedrock/issues/7707
Final copy is linked in https://github.com/mozilla/bedrock/issues/7706

**Comment by alexgibson** [comment]
Depends on https://github.com/mozilla/bedrock/pull/7757

**Comment by alexgibson** [comment]
WIP demo: https://www-demo1.allizom.org/en-US/firefox/privacy/products/

**Comment by alexgibson** [comment]
We're still waiting on some final imagery & design tweaks, as well as analytics instrumentation, but otherwise I think both pages are now far enough along to start internal review:

https://www-demo1.allizom.org/en-US/firefox/privacy/
https://www-demo1.allizom.org/en-US/firefox/privacy/products/

Note: to test the links that open `about:protections`, you'll need to whitelist the demo server to enable UITour. You can find information on how to do that [here](https://bedrock.readthedocs.io/en/latest/uitour.html#local-development).

**Comment by MKS-moz** [comment]
Thanks @alexgibson. Both pages are looking excellent. I see two things that might be of concern (or not), but I wanted to flag them for you:

1. “Learn more about Lockwise” currently links to a www-demo1.allizom.org page with a 404. I’m sure this will be updated, but wanted to call it out just in case. (To confirm, mozilla.org/firefox/lockwise will be live the same day or before this page will be live, yes? That’s the current intended URL to send folks to.)

2. When I tested signing up for Sync with a dummy email address on the form, I got this pop-up screen: https://www.screencast.com/t/dGtILPJWp. I found this to be a bit confusing as a user. It’s likely not the page form itself that caused this flow, but in the case it was, I’m calling it out here.

**Comment by alexgibson** [comment]
> “Learn more about Lockwise” currently links to a www-demo1.allizom.org page with a 404. I’m sure this will be updated, but wanted to call it out just in case. (To confirm, mozilla.org/firefox/lockwise will be live the same day or before this page will be live, yes? That’s the current intended URL to send folks to.)

The lockwise page is still in development in another PR, hence the 404. It will link once that page exists.

> When I tested signing up for Sync with a dummy email address on the form, I got this pop-up screen: https://www.screencast.com/t/dGtILPJWp. I found this to be a bit confusing as a user. It’s likely not the page form itself that caused this flow, but in the case it was, I’m calling it out here.

This message is not something under remit of the privacy page, and is a feature of Firefox Accounts. Testing the full FxA signup flow is not something that's straight forward to do on a demo server, and for the sake of providing feedback on that design of this page, I'm not sure it's something people need to test.

**Comment by alexgibson** [comment]
This is now awaiting final design assets.

**Comment by dzingeek** [comment]
@alexgibson  **Design Feedback Below:**
___________________________________
**Sub Nav**
- Can we underline or bold which section the user is on? For example, if a user is on the "our promise" page can we indicate that in the sub nab?
___________________________________
**Hero Section:**
Can we incorporate the zap into the hero text?
![Screen Shot 2019-09-30 at 2 28 21 PM](https://user-images.githubusercontent.com/3740746/65918464-91a6ce00-e38e-11e9-8682-0f442deba798.png)
___________________________________
**ETP Section:**
- Attached is an updated img for chart/graph
- Please use blue #00B3F4 for "See what Firefox has..." and "Update your Firefox...." CTAs
![Screen Shot 2019-09-30 at 2 27 52 PM](https://user-images.githubusercontent.com/3740746/65918485-a08d8080-e38e-11e9-8ed1-cd1dfecaa067.png)
- Change the second purple background color (more than...) to #45278D
![Screen Shot 2019-09-30 at 2 28 05 PM](https://user-images.githubusercontent.com/3740746/65918498-a71bf800-e38e-11e9-9c10-0914df0473ef.png)
- Left align images and copy for the three sections 
![Screen Shot 2019-09-30 at 2 29 17 PM](https://user-images.githubusercontent.com/3740746/65918526-b4d17d80-e38e-11e9-9868-66438e66f774.png)
- Illustrations should be coming your way tomorrow/wednesday
___________________________________
**Firefox Monitor Section:**
- attached is an updated img
___________________________________
**Firefox Lockwise Section:**
- Illustrations should be coming your way tomorrow/wednesday
___________________________________
**Firefox Send Section:**
- Illustrations should be coming your way tomorrow/wednesday
___________________________________
**Pocket Section:**
- looks good
___________________________________
**Firefox Account Section:**
- Can we make the box smaller? Like we have used on [/firefox/accounts](https://www.mozilla.org/en-US/firefox/accounts/)?
- Make headline copy 18px
___________________________________
**Read Privacy Notice Section:**
- Looks good
___________________________________
**Spacing Between sections (Fx monitor – FxA)**
Space between sections feels tight 
___________________________________
**Breakpoint 768**
- "More than 100,000,000"...text is too large, can we make it the next size down
- FxA form: is there anything we can do to fix how this looks. I worry about how hard this will be to read. Can we make it look more like the one on the right (making the headline copy smaller in the form)
![Screen Shot 2019-09-30 at 2 45 31 PM](https://user-images.githubusercontent.com/3740746/65919383-f95e1880-e390-11e9-94af-daa6f6bb27fb.png)
___________________________________
**Breakpoint 480**
- I don't think we need the graph image here...it makes the page longer and I think it's surves the user better to get them into the 3 talking points below quicker (see sketch file 480 artboard for reference)
- Can we lead with text instead of images for the monitor section – fxa section (see sketch file 480 artboard for reference)
- Space between sections (monitor – fxa) feels tight, too hard to tell where a section starts/ends
___________________________________
@alexgibson let me know if you have any questions/concerns

**Comment by alexgibson** [comment]
> Sub Nav
> Can we underline or bold which section the user is on? For example, if a user is on the "our promise" page can we indicate that in the sub nab?

Done, thanks

> Hero Section:
> Can we incorporate the zap into the hero text?

The zap doesn't yet exist in Protocol. Do we have code for it anywhere?

> Attached is an updated img for chart/graph

The attachment doesn't seem to be here?

> Please use blue #00B3F4 for "See what Firefox has..." and "Update your Firefox...." CTAs

Done thanks. I find the blue text on the purple background a little hard to read, but it does seem to get a pass in the a11y inspector in dev tools.

> Change the second purple background color (more than...) to #45278D

Done

> Left align images and copy for the three sections

I'll do this once you supply some images.

> Firefox Account Section:
> Can we make the box smaller? Like we have used on /firefox/accounts?
> Make headline copy 18px

Done

> Space between sections feels tight

Done

> "More than 100,000,000"...text is too large, can we make it the next size down

Done. This is overriding default typography sizing for the [Call Out component](https://protocol.mozilla.org/patterns/organisms/call-out.html). If you can funnel these types of requests into [component change requests](https://github.com/mozilla/protocol/issues/new/choose), then we can fix them upstream. Thanks

> FxA form: is there anything we can do to fix how this looks. I worry about how hard this will be to read. Can we make it look more like the one on the right (making the headline copy smaller in the form)

I've adjusted this the best I can given the layout.

> Breakpoint 480
> I don't think we need the graph image here...it makes the page longer and I think it's surves the user better to get them into the 3 talking points below quicker (see sketch file 480 artboard for reference)

Done

> Can we lead with text instead of images for the monitor section – fxa section (see sketch file 480 artboard for reference)

Done, thanks

> Space between sections (monitor – fxa) feels tight, too hard to tell where a section starts/ends

Done, thanks

**Comment by dzingeek** [comment]
@alexgibson attached are updated images + a few that were missing. Also, two more design tweaks if possible.
______________________
**480 breakpoint** 
- can we left align copy in the ink section? see image on the right for reference
![Screen Shot 2019-10-01 at 3 43 35 PM](https://user-images.githubusercontent.com/3740746/66006082-4bbb3a00-e462-11e9-8854-c517cfb54bbf.png)

______________________
**On other browsers** 

- Can we give the button and the text above a little more breathing room, they feel tight right now
- Bold the "Get Enhanced Tracking..." copy that leads to the download button 
- See image on the right for reference
![Screen Shot 2019-10-01 at 3 49 42 PM](https://user-images.githubusercontent.com/3740746/66006354-3c88bc00-e463-11e9-9643-cc7f6698c99d.png)


[privacy-product-imgs.zip](https://github.com/mozilla/bedrock/files/3678774/privacy-product-imgs.zip)

**Comment by alexgibson** [comment]
> can we left align copy in the ink section? see image on the right for reference

Done, thanks. Again, this is non-standard behavior for the hero component. Filing upstream [change requests issues](https://github.com/mozilla/protocol/issues/new/choose) is useful if this should be the new standard layout on small screens (or something that should be an option).

> Can we give the button and the text above a little more breathing room, they feel tight right now
> Bold the "Get Enhanced Tracking..." copy that leads to the download button

Done thanks

I've also updated the imagery.

**Comment by dzingeek** [comment]
sorry @alexgibson i have one updated svg..we had one small tweak 
[img-send.svg.zip](https://github.com/mozilla/bedrock/files/3682063/img-send.svg.zip)

**Comment by dzingeek** [comment]
@alexgibson here are some updated svgs this should be it for this page, thanks!
[privacy-etp-imgs.zip](https://github.com/mozilla/bedrock/files/3682550/privacy-etp-imgs.zip)

**Comment by ckinswood** [comment]
The top headline section with "Firefox protects your privacy in every product" is still so large it bumps much of the actual ETP content down page. The ETP content is the primary focus of the page and the original reason the page was scoped. How can we draw focus to the ETP section? I'd like to see smaller headline band or remove the line and start the page with ETP.

**Comment by MKS-moz** [comment]
@alexgibson the updated pages are looking wonderful. My final consideration for a tweak is this: As a user, I'm inclined to click on these three sections (see image), but they don't offer me click opportunities. I'm inclined to click on them because the copy offers me actions — "meet the trackers," "only one click away," and "see how we're helping you protect your info". Those actions can only be followed through on about:protections. 

Thoughts on making these images and headlines clickable for Firefox 70+ users, directed to about:protections? 

![image (5)](https://user-images.githubusercontent.com/55890448/66089917-3b34bd80-e54f-11e9-9bf9-a3da4c2d102e.png)

**Comment by alexgibson** [comment]
> The top headline section with "Firefox protects your privacy in every product" is still so large it bumps much of the actual ETP content down page. The ETP content is the primary focus of the page and the original reason the page was scoped. How can we draw focus to the ETP section? I'd like to see smaller headline band or remove the line and start the page with ETP.

I'll defer to @dzingeek here as this is part of the original, approved design. I do think the page needs a main headline from a structural point of view, as even though ETP is the main area of product focus, the page itself speaks to more than just Firefox the browser. I'll also point out here that we are past the cut off date for string changes, so whilst we could move the headline, or make it smaller, we can't change the copy.

**Comment by alexgibson** [comment]
> Thoughts on making these images and headlines clickable for Firefox 70+ users, directed to about:protections?

Done, thanks.

**Comment by alexgibson** [comment]
> @alexgibson here are some updated svgs this should be it for this page, thanks!
privacy-etp-imgs.zip

Updated, thanks.

**Comment by dzingeek** [comment]
> > The top headline section with "Firefox protects your privacy in every product" is still so large it bumps much of the actual ETP content down page. The ETP content is the primary focus of the page and the original reason the page was scoped. How can we draw focus to the ETP section? I'd like to see smaller headline band or remove the line and start the page with ETP.
> 
> I'll defer to @dzingeek here as this is part of the original, approved design. I do think the page needs a main headline from a structural point of view, as even though ETP is the main area of product focus, the page itself speaks to more than just Firefox the browser. I'll also point out here that we are past the cut off date for string changes, so whilst we could move the headline, or make it smaller, we can't change the copy.

I agree with @alexgibson. This page does need a main headline to give context to users so they know what else they can expect from this page. We could try and make that headline the next size down and see how it feels but I worry the headline will then get lost.

**Comment by dzingeek** [comment]
@alexgibson forgot to circle back on the zap in the headline. I know @stephaniehobson has been working on figuring out how to make this work. I don't think this needs to hold up the page. We can add it in after it goes live.

**Comment by hoosteeno** [comment]
@alexgibson 

* When I hover over the Fx-70 CTA ("See what Firefox has blocked for you") the URL preview at the bottom of the browser window suggests I'll go to SUMO, but instead I go to the protection report. Is it possible to change the href on this link to e.g "#" or similar so it isn't as confusing to people paying attention to that?
* The Pocket landing page we're sending people to from the Pocket link is a general-purpose landing page that does not drive FxA. Should we instead point that link at the Firefox-specific landing page? https://getpocket.com/firefox_learnmore. 
* Will the FxA form on this page pass through utm parameters from upstream?
* What's up with the "e" in "username" in the below picture? Does it look this way to you?
<img width="614" alt="Screen Shot 2019-10-03 at 1 53 10 PM" src="https://user-images.githubusercontent.com/94519/66159440-39b5d480-e5e5-11e9-8126-2b95e94a2bcd.png">

**Comment by dzingeek** [comment]
@hoosteeno nice catch on that "e" ! @alexgibson attached is an updated svg...somehow that "e"
[img-lockwise.svg.zip](https://github.com/mozilla/bedrock/files/3688045/img-lockwise.svg.zip)
 got filled in when I turned the text to outlines.

**Comment by alexgibson** [comment]
> When I hover over the Fx-70 CTA ("See what Firefox has blocked for you") the URL preview at the bottom of the browser window suggests I'll go to SUMO, but instead I go to the protection report. Is it possible to change the href on this link to e.g "#" or similar so it isn't as confusing to people paying attention to that?

The SUMO links are there as fallback incase there's an issue with UITour in someone's profile. That said, I can hide those URLs if UITour is working, replacing them with `#`, if we think it's worth it.

> The Pocket landing page we're sending people to from the Pocket link is a general-purpose landing page that does not drive FxA. Should we instead point that link at the Firefox-specific landing page? https://getpocket.com/firefox_learnmore.

Will update 👍 

> Will the FxA form on this page pass through utm parameters from upstream?

Nope that work has yet to be done. I'm waiting on code review for https://github.com/mozilla/bedrock/pull/7733 before we tackle that functionality.

> What's up with the "e" in "username" in the below picture? Does it look this way to you?

I'll update with the image @dzingeek provided, thanks
